### ISSUE TITLE
boxplot with min-max extent overrides transform

### ISSUE THREAD

**Issue by meetamit** [vega/vega-lite#4140]
Hi there. It appears that using a boxplot with `extent: 'min-max'` causes any existing `transform`s to be ignored (or overwritten or perhaps processed later than the boxplot's transforms). However, if `extent` is a number, `transform` is *correctly* processed.

To demonstrate, in the following spec we `calculate` variable `b2` and use it as `field` in a 1D boxplot with `extent: min-max`. This does **not** render correctly (Infinite domain). However, changing to `extent: 100000`, which (for this dataset) produces the same result as `min-max`, calculates and renders properly. 

````json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v3.0.0-rc3.json",
  "data": {
    "values": [
      {"a": "A", "b": 28},
      {"a": "B", "b": 55},
      {"a": "C", "b": 43},
      {"a": "G", "b": 19},
      {"a": "H", "b": 87},
      {"a": "I", "b": 52},
      {"a": "D", "b": 91},
      {"a": "E", "b": 81},
      {"a": "F", "b": 53}
    ]
  },
  "transform": [
    {"calculate": "2*datum.b", "as": "b2"}
  ],
  "mark": {
    "type": "boxplot",
    "extent": "min-max"
  }, 
  "encoding": {
    "y": {"field": "b2", "type": "quantitative"}
  }
}
````
Thanks!

**Comment by domoritz** [comment]
Thank you for reporting the bug. It looks like a bug in the normalizer code. We will look into it.

**Comment by domoritz** [comment]
Fixed in #4141
